core/lf_token.c

```diff
@@ -352,8 +367,7 @@ token_freed _lf_done_using(lf_token_t* token) {
 
 void _lf_free_token_copies() {
   while (_lf_tokens_allocated_in_reactions != NULL) {
-    lf_token_t* next = _lf_tokens_allocated_in_reactions->next;
```

Comment:
Looking at `_lf_done_using` I cannot see why this reordering makes any difference?

Reply:
You are right, this is not the fix for the bug. This is just a cleanup. In fact, all the changes in this file are just cleanups.  The fix is in the lingua-franca PR.